subcommand/lifecycle-sidecar/command.go

```diff
@@ -34,6 +41,7 @@ type Command struct {
 func (c *Command) init() {
 	c.flagSet = flag.NewFlagSet("", flag.ContinueOnError)
 	c.flagSet.StringVar(&c.flagServiceConfig, "service-config", "", "Path to the service config file")
+	c.flagSet.StringVar(&c.flagConsulLocation, "consul-location", "", "Path to a consul binary")
```

Comment:
I think this should be `-consul-binary` to match the flag from `consul connect envoy`:
```
  -envoy-binary=<string>
     The full path to the envoy binary to run. By default will just
     search $PATH. Ignored if -bootstrap is used.
```

Reply:
Renamed.